rubocop-rspec_rails.gemspec

```diff
@@ -34,5 +34,6 @@ Gem::Specification.new do |spec|
     'rubygems_mfa_required' => 'true'
   }
 
+  spec.add_runtime_dependency 'rack'
```

Comment:
For the time being, we’ll probably have `rubocop-rspec_rails` being a runtime dependency of `rubocop-rspec`, just like `rubocop-capybara` and `rubocop-factory_bot` currently are: https://github.com/rubocop/rubocop-rspec/blob/447fa4e3d63e8998c8039ff7b9a2b216f61aa35c/rubocop-rspec.gemspec#L41-L42

And we don’t want `rack` to be a transitive runtime dependency of rubocop-rspec.

Reply:
Ah, I see. My apologize. I revert it.